tools/tertiary-analysis/monocle3/monocle3-topmarkers.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+<tool id="monocle3_topmarkers" name="Monocle3 top markers" version="0.1.5+galaxy0" profile="@PROFILE@">
+  <description>for cell groups</description>
+  <macros>
+    <import>monocle3-macros.xml</import>
+  </macros>
+  <requirements>
+    <requirement type="package" version="0.0.8">monocle3-cli</requirement>
+  </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    ln -s '$input_object' input.rds;
+    #if $ref_cells.source == 'file' and $ref_cells.file
+    list_of_cells = \$(cat $ref_cells.file | tr '\n' ',')
```

Comment:
I don't know exactly how code here is handled by galaxy, but if it's translated to bash, then perhaps space around the `=` needs to be removed? This is of course assuming this code hasn't been tested.